package.json

```diff
@@ -3,6 +3,7 @@
   "description": "Developer tools for Apollo Client, with Studio Explorer integration, cache inspection, and query watching.",
   "license": "MIT",
   "version": "4.9.0",
+  "type": "module",
```

Comment:
As a result, this is now a module, and I had to generally change a few files around.